app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<section class="form-signup">
+  <h2 class='title'>Sign up</h2>
+
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+    <div class="field">
+      <%= f.label :name %>
+      <%= f.text_field :name%>
+    </div>
+    <div class="field">
+      <%= f.label :bio %>
+      <%= f.text_area :bio%>
+    </div>
+    <div class="field">
+      <%= f.label :email %>
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password %>
+      <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %>
+      <%= f.password_field :password, autocomplete: "new-password" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password_confirmation %>
+      <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+    </div>
+
+  <div class="field">
+      <%= f.label :Profile_picture %>
+      <%= f.file_field :photo%>
+    </div>
+
+    <div class="actions">
+      <%= f.submit "Sign up" %>
+    </div>
+  <% end %>
```

Comment:
- [ ] When I click on the signup button, I don't see a message that I have successfully signed up I see an error  `(6 characters minimum)`  and my passwords more than 6 but when I click to confirm my email and I signin I can see the user I created please clear the error, so that when users click to submit the signup form they are redirected to the home page where they see the user created, see video below ⬇️ 

https://user-images.githubusercontent.com/69638013/209294115-c2707281-c9c1-4da9-b65a-d8b2ddbeb88a.mp4